.github/workflows/tests.yml

```diff
@@ -0,0 +1,81 @@
+name: Tests
+
+on:
+    push:
+    pull_request:
+
+jobs:
+
+    integration:
+        name: Integration
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                php: ['5.5', '7.2', '7.3', '7.4']
+
+        services:
+            redis:
+                image: redis
+                ports:
+                    - 6379:6379
+            redis-cluster:
+                image: grokzen/redis-cluster:4.0.8
+                ports:
+                    - 7000:7000
+                    - 7001:7001
+                    - 7002:7002
+                    - 7003:7003
+                    - 7004:7004
+                    - 7005:7005
+            memcached:
+                image: memcached:1.6.5
+                ports:
+                    - 11211:11211
+
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    coverage: "none"
+                    extensions: "redis,memcached"
+                    ini-values: "memory_limit=-1"
+                    php-version: "${{ matrix.php }}"
+
+            -   name: Configure composer
+                run: |
+                    ([ -d ~/.composer ] || mkdir ~/.composer) && cp .github/composer-config.json ~/.composer/config.json
+                    SYMFONY_VERSION=$(cat composer.json | grep '^ *\"dev-master\". *\"[1-9]' | grep -o '[0-9.]*')
+                    echo "::set-env name=SYMFONY_VERSION::$SYMFONY_VERSION"
+                    echo "::set-env name=COMPOSER_ROOT_VERSION::$SYMFONY_VERSION.x-dev"
+
+            -   name: Determine composer cache directory
+                id: composer-cache
+                run: echo "::set-output name=directory::$(composer config cache-dir)"
+
+            -   name: Cache composer dependencies
+                uses: actions/cache@v1
+                with:
+                    path: ${{ steps.composer-cache.outputs.directory }}
+                    key: ${{ matrix.php }}-composer-${{ hashFiles('**/composer.lock') }}
+                    restore-keys: ${{ matrix.php }}-composer-
+
+            -   name: Install Symfony Flex
+                run: composer global require --no-progress --no-scripts --no-plugins symfony/flex dev-master
+                if: matrix.php != 5.5 && matrix.php != 5.6
```

Comment:
I just noted that 5.6 is not in the matrix.

Reply:
Thanks for noting. I removed some versions for now until we agree which versions should be included in the matrix.

Reply:
Forgot to remove the condition. Will do it next! :)